src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -301,7 +301,6 @@ package com.videojs.providers{
                 // if the asset is already loading
                 if (_hasEnded) {
                   _hasEnded = false;
-                  _ns.seek(0);
```

Comment:
Looping continues to work fine.

I'm not sure when this code is actually being used, as it doesn't seem to be interfering with anything, but since we've decided to let the player (not the tech) decide when/whether to reset, this should still go.